.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,22 @@
+name: PR
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  soundness:
+    name: Soundness
+    uses: swiftlang/github-workflows/.github/workflows/soundness.yml@main
+    with:
+      license_header_check_project_name: "RediStack"
+
+  unit-tests:
+    name: Unit tests
+    uses: apple/swift-nio/.github/workflows/unit_tests.yml@main
```

Comment:
Do we want to use the swift lang org workflow instead?

Reply:
Yeah that's a good point

Reply:
Hm looking at various repos we have a mix of which ones we use... I may stick with this for now, we'll see.